template/page-studs/index/with-auth-trpc-tw.tsx

```diff
@@ -42,62 +41,74 @@ const Home: NextPage = () => {
 
         <h3 className="items-center m-5 text-3xl">This Stack uses:-</h3>
 
-        <section className="grid items-start grid-cols-1 gap-10 p-5 md:p-0 sm:grid-cols-1 md:grid-cols-2 xl:grid-cols-3 2xl:grid-cols-5 ">
-          <section className="p-5 py-10 text-center duration-500 transform border-2 cursor-pointer border-neutral-800 rounded-2xl hover:scale-105 hover:border-blue-600 hover:-translate-y-1 hover:shadow-2xl">
-            <h1 className="my-5 text-3xl">NextJS</h1>
-            <p className="mb-5">The React framework for production</p>
-            <button className="p-2 px-6 text-white bg-blue-500 rounded-md hover:bg-blue-600">
-              <Link href="https://nextjs.org/">
-                <a target="_blank">Learn More</a>
-              </Link>
-            </button>
+        <main className="grid items-start grid-cols-1 gap-10 p-5 md:p-0 sm:grid-cols-1 md:grid-cols-2 xl:grid-cols-3 2xl:grid-cols-5 ">
+          
+          <section className="h-full transform group border-2 border-neutral-800 rounded-2xl duration-300 hover:scale-105 hover:border-blue-600 hover:-translate-y-1 hover:shadow-2xl">
+            <a href="https://nextjs.org/" target="_blank" rel="noreferrer">
+              <div className="p-5 py-10 flex flex-col justify-center h-full cursor-pointer text-center">
+                <h1 className="mb-5 text-3xl">NextJS</h1>
+                <p className="mb-5">The React framework for production</p>
+                <button className="p-2 px-6 w-fit self-center text-white font-bold bg-blue-500 rounded-full group-hover:bg-blue-600 duration-300">
+                  Documentation
+                </button>
+              </div>
+            </a>
           </section>
 
-          <section className="p-5 py-10 text-center duration-500 transform border-2 cursor-pointer border-neutral-800 rounded-2xl hover:scale-105 hover:border-blue-600 hover:-translate-y-1 hover:shadow-2xl">
-            <h1 className="my-5 text-3xl">TypeScript</h1>
-            <p className="mb-5">
-              Strongly typed programming language that builds on JavaScript,
-              giving you better tooling at any scale
-            </p>
-            <button className="p-2 px-6 text-white bg-blue-500 rounded-md hover:bg-blue-600">
-              <Link href="https://www.typescriptlang.org/">
-                <a target="_blank">Learn More</a>
-              </Link>
-            </button>
+          <section className="h-full transform group border-2 border-neutral-800 rounded-2xl duration-300 hover:scale-105 hover:border-blue-600 hover:-translate-y-1 hover:shadow-2xl">
```

Comment:
Is `hover:-translate-y` correct? The extra - at the beginning in particular

Reply:
I believe so - its just a negative value in tailwind, i.e. negative margin on hover would be `hover:-mt-8`. It's from the initial tailwind styles, not my code so I'm not 100% confident.